azure-pipelines.yml

```diff
@@ -17,5 +30,7 @@ steps:
 
 - script: |
     npm install
+    npm run lint
     npm run build
-  displayName: 'npm install and build'
+    npm run test
+  displayName: 'npm install, lint, build and test'
```

Comment:
Generally prefer having these split across multiple steps so its easier to see at a glance whats failing
```yaml
- script: npm install
- script: npm run lint
- script: npm run build
- script: npm run test
```

Reply:
Yep, this!